test/e2e/pkg/manifest.go

```diff
@@ -178,6 +178,9 @@ type ManifestNode struct {
 	// It defaults to false so unless the configured, the node will
 	// receive load.
 	SendNoLoad bool `toml:"send_no_load"`
+
+	// Geographical zone ID for simulating latencies.
+	Zone string `toml:"zone"`
```

Comment:
What happens with nodes that don't have a zone set? (my understanding is that the node doesn't get any latency set)
Does is have a valid physical interpretation?
Should we just set the no zone nodes to a default zone? (so latencies are always on, whether the user wants them or not)
Or should there be a general boolean to enable/disable latencies, and if enabled, nodes with no zone would go to a default zone?

Reply:
A node with no zone will have 0 added latency when sending or receiving messages to other nodes. It's like it belongs to a zone that is close to all zones, if that makes any sense. I don't know if this should be allowed or not, but it doesn't do any harm, except maybe when you forget to assign a zone to a node.

Reply:
(sorry for delay getting back to this) I think it makes sense to define a default behavior that does make sense (a zone close to all others is a bit of a stretch IMO)

Reply:
I added a `default_zone` parameter to the testnet. Now either all nodes have a zone or none have.